lib/pwa.js

```diff
@@ -278,12 +278,10 @@ exports._save = function(pwa) {
     })
     .then(savedPwa => {
       this.addPwaToCache(savedPwa);
-      this.updateIcon(savedPwa, pwa.manifest)
+      this.updateIcon(savedPwa, pwa.manifest);
+      this.updateLighthouseInfo(savedPwa)
```

Comment:
The reason that updateLighthouseInfo is called on the .then part of updateIcon is to guarantee the order in which the steps are executed. Is this change needed to make it work? Or the code below alone should work?